client/packages/coldchain/src/Equipment/ImportAsset/ImportTab.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
+import { InlineProgress, Typography } from '@openmsupply-client/common';
+import { ImportPanel } from './ImportPanel';
+import { useTranslation } from '@common/intl';
+import { Grid } from '@openmsupply-client/common';
+
+interface EquipmentImportTabProps {
+  importProgress: number;
+  importErrorCount?: number;
+}
+
+export const EquipmentImportTab: FC<ImportPanel & EquipmentImportTabProps> = ({
+  tab,
+  importProgress,
+  importErrorCount,
+}) => {
+  const t = useTranslation('coldchain');
+  return (
+    <ImportPanel tab={tab}>
+      <Grid
+        container
+        direction="column"
+        justifyContent="center"
+        style={{ minHeight: '50vh' }}
+      >
+        <Typography>
+          {importErrorCount ?? 0 > 0
+            ? '(' +
+              t('messages.error-generic', { count: importErrorCount }) +
+              ')'
+            : ''}
```

Comment:
:nit the + and multiline formatting is unusual. would you mind this?
```suggestion
          {importErrorCount ?? 0 > 0
            ? `(${t('messages.error-generic', { count: importErrorCount })})`
            : ''}
```